pubspec.yaml

```
@@ -7,10 +7,10 @@ funding:
   - https://github.com/sponsors/schultek
 
 environment:
-  sdk: '>=2.17.0 <3.0.0'
+  sdk: '>=2.17.0 <4.0.0'
```

I think its ok to bump this to >=3.0.0
```suggestion
  sdk: '>=3.0.0 <4.0.0'
```

---

I also updated the `lints` and `file` packages.